### ISSUE TITLE
"Unknown structure start: awesome_dungeon:the_z_tower" log spam 1.16.5

### ISSUE THREAD

**Issue by Ryuushen** [jtorleonstudios/HUB-Awesome-Dungeon#14]
![cmd_1QFmj9YEIC](https://user-images.githubusercontent.com/24853902/225084024-dec4ea1a-6e37-4676-ac64-900a5eb9035b.png)
Log spam on 1.16.5 Forge server after updating LibraryFerret and all the Awesome dungeon yesteday

**Comment by jtorleonstudios** [comment]
Yes, the identifier of the structure to be changed for various internal reasons.  (old id: `the_z_tower` , new id: `the_ztower`)

The error is launched because Minecraft does not find the old structure, it is a bug on the side of mojang. (see: [https://bugs.mojang.com/browse/MC-194811](https://bugs.mojang.com/browse/MC-194811))

To stop spam you can use a new world
or clean up the references with the available tools  (example: [MCStructureCleaner](https://github.com/Nyveon/MCStructureCleaner), NbtExplorer etc...).

I am sorry in the inconvenience created by this modification, but it is noted that is extremely rare that we change the identifiers of the structures, on 2 years we have done it 2 times, it should not happen anymore in the future.

Thanks you for your report.
